lib/danger/danger_core/plugins/dangerfile_github_plugin.rb

```diff
@@ -224,6 +224,16 @@ def html_link(paths, full_path: true)
       paths.first(paths.count - 1).join(", ") + " & " + paths.last
     end
 
+    # @!group Github Misc
+    # Tell github to ingore any inline message whihc is not in diff's range, not post as main comment.
+    # @param    [Bool] dismiss
+    #           Ingore out of range inline messages, defaults to `true`
+    #
+    # @return   [void]
+    def dismiss_out_of_range_messages(dismiss: true)
```

Comment:
Please make this default to false, I would expect that to be the most common experience. I would always want to know all the warnings/errors to force people to actually fix them ;)